tests/test_trainer.py

```diff
@@ -57,7 +57,7 @@ def check_train_on_catdog_datasets(trainer_args={}, expected_model_spec={}, expe
               'model_kwargs': {'alpha': 1.0},
               'momentum': 0.9,
               'num_classes': 2,
-              'num_gpus': 1,
+              'gpu_ids': [0],
```

Comment:
Not sure why I set this as the default... but you can delete it, otherwise CI won't run